pennylane/fermi/fermionic.py

```diff
@@ -158,11 +160,15 @@ def __pow__(self, value):
 
         return operator
 
+    def to_qubit(self):
+        """Map the FermiWord to the qubit basis to get a linear combination
+        of Pauli operators"""
+        return qml.fermi.jw_mapping(self)
+
```

Comment:
I'm not sure if we need a method here. Could we just point the user to `qml.fermi.to_qubit(my_fermi_word)`?

Reply:
@trbromley, we can have both options?

Reply:
I imagine this function will have more logic when we add other mapping methods?

Reply:
@Jaybsoni Yes, it should have more logic if we add more mapping methods.

Reply:
I'm not sure if this case would specifically be a problem, but it's best to stick to pure functions where possible for the JIT pipelines.

Reply:
We can safely remove the `to_qubit` methods in `FermiWord` and `FermiSentence`.

Reply:
I will be shocked if the `jordan_wigner` function is jittable, but I can remove `to_qubit` if that's the preference. 

Is the plan for this to be JIT compatible with jax?